OpenRA.Mods.Common/InstallUtils.cs

```diff
@@ -39,11 +39,8 @@ public class ContentInstaller : IGlobalModData
 		public readonly string MusicPackageMirrorList = null;
 		public readonly int ShippedSoundtracks = 0;
 
-		/// <summary> InstallShield .cab File Ids, used to extract Mod specific files </summary>
-		public readonly int[] InstallShieldCABFileIds = { };
-
-		/// <summary> InstallShield .cab File Ids, used to extract Mod specific archives and extract contents of ExtractFilesFromCD </summary>
-		public readonly string[] InstallShieldCABFilePackageIds = { };
+		public readonly string InstallShieldCABDirectory = null;
+		public readonly string[] FiletoExtractFromCab = { };
```

Comment:
Please keep with the `InstallShieldCAB*` naming convention set for these.
Also don't remove the `<summary>`-ies.

Oh, and the ` = null` is not necessary.